.travis.yml

```diff
@@ -7,8 +7,6 @@ before_script:
 - chmod +x ../scripts/travisTest.sh
 script: 
 - ../scripts/travisTest.sh
-- serverName=$(target/liberty/wlp/bin/server list | cut -d '.' -f2| tr -d '\n')
```

Comment:
Actually I checked that the other guide, ie, cdi-intro (ref https://github.com/OpenLiberty/guide-cdi-intro/blob/master/.travis.yml), does have these steps to collect the runtime logs and send it to the artifactory after a PR run (basically having these steps define in the `.travis.yml` file. Perhaps it's good to keep it consistent, provided that they're working properly for this guide.